src/components/App.tsx

```diff
@@ -51,13 +51,14 @@ export const App = () => {
       <DateRange />
       <div className="divider" />
       <AttributesSelector />
+      {state.attributes.length > 0 && <AttributeFilter />}
       <div className="divider" />
       <div className="footer">
         <button className="clear-data-button">Clear Data</button>
         <button className="get-data-button">Get Data</button>
       </div>
       {showModal === "info" && <InfoModal />}
-      <StateCounterDemoToBeRemoved />
+      {showModal === "data-return-warning" && <DataReturnWarning />}
```

Comment:
I know @lublagg also did some work on the data return warning ui - does this conflict with that?